### PR TITLE
PPSS-40347 Update readme file with FAQ and release notes for Application Browser

### PR DESCRIPTION
**Pull request by caeduroa**
Update readme file with FAQ and release notes for Application Browser

### CODE REVIEW

**Review by curlyleung — COMMENTED** [review]
Few comments:

- The FAQ.docx should be translated to a markdown file in the repository and the README.md should link to the FAQ markdown.  Should not be a new asset.
- Same comment for CHANGES.txt.  Should this be a markdown file?   
- Can you verify the contents for the CHANGES (or call it Release Notes instead?) with Robert?
-  The main repo’s README can have a prominent message that its contents apply to the “current version” and that users should browse the tags in Github to see the contents for any other version(s).

File: RELEASENOTES.md [hunk 1]
```diff
@@ -1,5 +1,120 @@
...
+
+•	Run the “ApplicationBrowser-<version>.msi” file.
+- While the install in running, a status window will display.
+
+•	Once installation is complete:
+- ApplicationBrowser will be installed under C:\Program Files (x86)\Application Browser\
+- Shortcuts should now be available on the windows start menu. You may also want to right-click this icon in the application bar to pin it.
+- You will need to configure the application environments. The application will prompt the user if the configuration file is missing or is invalid.
+
+### Using Application Browse exe artifact
```
Comment (curlyleung) [inline RELEASENOTES.md]: Misspelled "Browser"

File: RELEASENOTES.md [hunk 2]
```diff
@@ -1,5 +1,120 @@
...
+      "endsWith": "PPSS/PPSS",
+      "thenAppend": "?control=loginPage"
+    }
+  ]
+}
+```
+
+* flash > path (optional) - The path to the installed pepper flashplayer.dll file. The application will attempt to use this path to the Pepper Flash Player. If not found, the application will attempt to find an installed version.
+* environments (required) - An array of environments that will be available on the initial page. Each environment has the following properties:
+* id (required) – A unique id for the environment.
```
Comment (curlyleung) [inline RELEASENOTES.md]: id, url, and label should be indented under environments.
image

File: RELEASENOTES.md [hunk 3]
```diff
@@ -1,5 +1,120 @@
...
+* label (required) - The name of the link that will be shown in the UI. This label is also used in the title upon opening an environment
+* develop (optional) - A Boolean value used to turn on development mode. This is not recommended for production installations.
+* locale (optional) – A locale value that will override the system locale. This is not recommended for production installations.
+* whenUrl (optional) – A set of conditions used to update a URL when opened from the main window.
+	* endsWith – A string to look for in the URL when loaded
+	* thenAppend – A string to append to the end of a URL if the endsWidth string is found.
+After the application is successfully opened, it creates the launchpage-config.json file in the user’s folder. This file has the hide property for each configured environment that indicates if this environment is hidden or not. The user will be able to override this value within the running application. Here is a sample of this file:
```
Comment (curlyleung) [inline RELEASENOTES.md:96]: This should be new paragraph, right now it is appended to thenAppend.
image

File: RELEASENOTES.md [hunk 4]
```diff
@@ -1,5 +1,120 @@
...
+         "ref":"sandbox",
+         "hide":false
+      },
+      {
+         "ref":"production",
+         "hide":false
+      }
+   ]
+}
+```
```
Comment (curlyleung) [inline RELEASENOTES.md:114]: Missing "Configuring PPSS with a "Load Balancer", "Deep Linking", "Launch the Application Browser with an ID parameter", and "How to Configure PPSS DO Email Approval" sections?

**Comment by curlyleung** [comment]
For the statement "The contents of this document apply to the “current version”. Browse the release tags in Github to see the contents for any other version(s).", can we use "latest release" instead of "current version"?

**Review by curlyleung — COMMENTED** [review]
Looks good to me.

**Review by curlyleung — APPROVED** [review]
Approved.

**Review by kevinmprince — APPROVED** [review]

**Comment by kevinmprince** [comment]
Approved, please confirm the release version number is correct. I don't know if you need to change it since this is just a doc update.

**Comment by caeduroa** [comment]
This changes will be introduced in a new PR for release 4.0.2

**Pull request closed without merging**